assets/report_submitted.svg

```diff
@@ -0,0 +1,11 @@
+<svg

```

Comment:
This isn't an SVG? Use the original SVG, or if you really cannot find it, just a PNG file.

Reply:
I'm unable to find the original SVG file by discord. Should I use this PNG: https://cdn.discordapp.com/attachments/904385235649691728/921070718622519347/unknown.png

Reply:
@CanadaHonk Changed it to a PNG as im unable to find the original SVG by discord. Let me know if I should change anything to this.